app/src/main/res/layout/item_skill.xml

```diff
@@ -77,26 +78,40 @@
                 android:textSize="@dimen/text_size_moderate"
                 android:fontFamily="sans-serif"/>
 
-            <LinearLayout
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="horizontal">
 
                 <RatingBar
                     android:id="@+id/cv_rating_bar"
                     style="?android:attr/ratingBarStyleSmall"
                     android:layout_width="@dimen/rating_bar_layout_width"
                     android:layout_height="wrap_content"
+                    android:layout_gravity="center"
                     android:numStars="5"
                     android:padding="@dimen/cmv_padding"
                     android:theme="@style/RatingBar" />
```

Comment:
Indentation of the RatingBar is wrong

Reply:
So it should be left aligned?

Reply:
It should be one indent less. Please correct me if I am wrong

Reply:
It is looking fine for now, can be shifted to left when average stars will be shown along with rating. Pls give your views.